File: TripKitAndroidUI/src/main/java/com/skedgo/tripkit/ui/core/module/TimePickerComponent.java
package com.skedgo.tripkit.ui.core.module;

import com.skedgo.tripkit.ui.dialog.TripKitDateTimePickerDialogFragment;
import dagger.Subcomponent;

@ActivityScope
@Subcomponent(modules = TimePickerModule.class)
public interface TimePickerComponent {
  void inject(TripKitDateTimePickerDialogFragment fragment);
}

File: TripKitAndroidUI/src/main/java/com/skedgo/tripkit/ui/core/module/TimePickerComponent.kt
package com.skedgo.tripkit.ui.core.module;

import com.skedgo.tripkit.ui.dialog.TripKitDateTimePickerDialogFragment;
import com.skedgo.tripkit.ui.dialog.v2.datetimepicker.TKUIDateTimePickerDialogFragment;

import dagger.Subcomponent;

@ActivityScope
@Subcomponent(modules = TimePickerModule.class)
public interface TimePickerComponent {
    void inject(TripKitDateTimePickerDialogFragment fragment);
    void inject(TKUIDateTimePickerDialogFragment fragment);
}